test/capture.js

```diff
@@ -1,3 +1,21 @@
+define([ 'intern!tdd',
+        'intern/chai!expect',
+        '../pointerevents',
+        'src/boot',
+        'src/pointermap',
+        'src/capture',
+        'src/dispatcher',
+        'src/installer',
+        'src/mouse',
+        'src/ms',
+        'src/platform-events',
+        'src/PointerEvent',
+        'src/touch',
+        'src/touch-action'
+       ],
+       function (tdd, expect) {
+           with (tdd) {
```

Comment:
That's the way the intern docs show it, and it doesn't seem to work without it.  Do you have an alternative?